### DISCUSSION TITLE
Is there any way to open Mountain Duck in a window and not as a popup?

### DISCUSSION THREAD

**Discussion by ThorstenDtk** [iterate-ch/cyberduck#16259] [Q&A]
I didn't find any option  to open Mountain Duck in a window and not as a popup (on Windows). Is that possible somehow?

**Comment by dkocher** [comment]
Mountain Duck is opened as an application in the taskbar only and there is no preference to make it open as a regular application with a window instead.

**Reply by ThorstenDtk** [reply]
OK, thanks for letting me know. That would be a nice feature, especially when syncing a lot of data as it would allow you to monitor the progress and not having to open the pop up frequently.
